modules/ROOT/pages/appendix/gql-conformance/additional-cypher.adoc

```diff
@@ -507,29 +507,17 @@ GQL supports `GRAPH TYPES` as a way of constraining a graph schema, but does not
 | Cypher feature
 | Description
 
-| xref:constraints/examples.adoc#constraints-examples-node-uniqueness[Node property uniqueness constraints]
-| Ensures that certain nodes have a set of specified properties whose combined value is unique when all properties exist on the node
+| xref:constraints/managing-constraints.adoc#create-uniqueness-constraints[Uniqueness constraints]
+| Ensure that property values are unique for all nodes with a specific label or all relationships with a specific type. 
 
-| xref:constraints/examples.adoc#constraints-examples-relationship-uniqueness[Relationship property uniqueness constraints]
-| Ensures that certain relationships have a set of specified properties whose combined value is unique when all properties exist on the relationship.
+| xref:constraints/managing-constraints.adoc#create-existence-constraints[Existence constraints]
+| Ensure that a property exists either for all nodes with a specific label or for all relationships with a specific type.
 
-| xref:constraints/examples.adoc#constraints-examples-node-property-existence[Node property existence constraints]
-| Ensures that certain nodes have a specified property.
+| xref:constraints/managing-constraints.adoc#create-type-constraints[Type constraints]
+| Ensure that a property has the required property type for all nodes with a specific label or for all relationships with a specific type.
 
-| xref:constraints/examples.adoc#constraints-examples-relationship-property-existence[Relationship property existence constraints]
-| Ensures that certain relationships have a specified property.
-
-| xref:constraints/examples.adoc#constraints-examples-node-property-type[Node property type constraints]
-| Ensures that certain nodes have a property of the required property type when the property exists on the node.
-
-| xref:constraints/examples.adoc#constraints-examples-relationship-property-type[Relationship property type constraints]
-| Ensures that certain relationships have a property of the required property type when the property exists on the relationship.
-
-| xref:constraints/examples.adoc#constraints-examples-node-key[Node key constraints]
-| Ensures that certain nodes have a set of specified properties whose combined value is unique and all properties in the set are present.
-
-| xref:constraints/examples.adoc#constraints-examples-relationship-key[Relationship key constraints]
-| Ensures that certain relationships have a set of defined properties whose combined value is unique. It also ensures that all properties in the set are present.
+| | xref:constraints/managing-constraints.adoc#create-key-constraints[Key constraints]
+| ensure that, for a given node label or relationship type and set of properties, all properties are present on the respective nodes or relationships, and the combination of property values is unique.
```

Comment:
```suggestion
| xref:constraints/managing-constraints.adoc#create-key-constraints[Key constraints]
| Ensure that, for a given node label or relationship type and set of properties, all properties are present on the respective nodes or relationships, and the combination of property values is unique.
```
also why is this ones description so different from the others?
```suggestion
| xref:constraints/managing-constraints.adoc#create-key-constraints[Key constraints]
| Ensure that property values are unique and exists for all nodes with a specific label or all relationships with a specific type. 
```
would be more consistent, wouldn't it? (though the comment from the property uniqueness constraints will then apply here as well)